samples/webview-samples/multi-tab-browser/tabs.js

```diff
@@ -268,36 +272,13 @@ var tabs = (function(popupModule, contextMenuModule) {
     this.loading = false;
   };
 
-  Tab.prototype.doContentLoad = function(e) {
-    if (!this.scriptInjectionAttempted) {
-      // Try to inject title-update-messaging script
-      (function(tab) {
-        tab.webview.addContentScripts([
-          {
-            'name': 'Messageing',
-            'matches': ['<all_urls>'],
-            'js': { 'files': ['guest_messaging.js']},
-            'run_at': 'document_end'
-          }]);
-      }(this));
-      this.scriptInjectionAttempted = true;
-    }
-  };
-
-  Tab.prototype.doScriptInjected = function(results) {
-    if (!results || !results.length) {
-      console.warn(
-          'Warning: Failed to inject guest_messaging.js',
-          this.webview);
-    } else {
-      // Send a message to the webview so it can get a reference to
-      // the embedder
-      var data = {
-        'type': 'titleRequest',
-        'tabName': this.name
-      };
-      this.webview.contentWindow.postMessage(JSON.stringify(data), '*');
-    }
+  Tab.prototype.doContentLoad = function(e) {    
+    // TODO: add the code which is to be run after 'contentload' here.
```

Comment:
Not anymore. I removed it and so did with the comment since it was somewhat obvious.